src/components/SavedSongs/SavedSongList.js

```diff
@@ -12,36 +13,61 @@ SavedSongList.propTypes = {
 }
 
 export default function SavedSongList({
-  stopPlayingSong,
+  stopPlayingSongById,
   deleteSavedSong,
   savedSongs,
   toggleCurrentSongId,
   isSongPlaying,
   currentSongId,
 }) {
   return (
-    <List>
-      {savedSongs?.map(({ artist, songTitle, id }) => (
-        <SavedSong
-          key={id}
-          artist={artist}
-          songTitle={songTitle}
-          id={id}
-          stopPlayingSong={stopPlayingSong}
-          deleteSavedSong={deleteSavedSong}
-          toggleCurrentSongId={toggleCurrentSongId}
-          isSongPlaying={isSongPlaying}
-          currentSongId={currentSongId}
-        />
-      ))}
-    </List>
+    <>
+      <HeaderFixed>Favorites</HeaderFixed>
+      <List>
+        {savedSongs?.map(({ artist, songTitle, id }) => (
+          <SavedSong
+            key={id}
+            artist={artist}
+            songTitle={songTitle}
+            id={id}
+            stopPlayingSongById={stopPlayingSongById}
+            deleteSavedSong={deleteSavedSong}
+            toggleCurrentSongId={toggleCurrentSongId}
+            isSongPlaying={isSongPlaying}
+            currentSongId={currentSongId}
+          />
+        ))}
+      </List>
+    </>
   )
 }
 
+const HeaderFixed = styled(Header)`
+  top: 0;
+  left: 0;
+  position: fixed;
+`
 const List = styled.ul`
   display: grid;
-  gap: 8px;
+  gap: 32px;
   list-style: none;
   margin: 0;
-  padding: 0;
+  overflow-y: auto;
+  padding: 0 0;
+  scrollbar-width: none;
+  &:before {
```

Comment:
I have written``::``for these pseudo elements. I think both is correct. ``::`` could be a bit more modern? Found this on [mozilla](https://developer.mozilla.org/en-US/docs/Web/CSS/::before)